apis/nucleus/src/index.js

```diff
@@ -157,6 +157,8 @@ const DEFAULT_CONTEXT = /** @lends Context */ {
    * @type {string=}
    * */
   dataViewType: 'sn-table',
+  /** @type {object=} */
```

Comment:
This should be of type Navigation? (from navigation.js)

Reply:
It should be navigation from sense client [navigation.js](https://github.com/qlik-trial/sense-client/blob/main/web/assets/objects/extension/supernova-wrapper/navigation.js), not sheet navigation from nebula which has less functions. 
I am not sure if we should write all the functions from sense in here, which we will maintain when sense client navigation changes, and if we want to expose what functions we have? I saw for example hostConfig doesn't specify, maybe a different case?

Reply:
I think it should always be typed as Navigation from Nebula, as that is what Nebula explicitly supports. Then it can be a different object but the required properties on that object is what Nebula defines.

Reply:
I added an objectnavigation type